lib/delighted/http_response.rb

```diff
@@ -37,5 +42,17 @@ def get_header_value(key)
         end
       end
     end
+
+    def parse_link_header(header_value)
+      links = {}
+      # Parse each part into a named link
+      header_value.split(',').each do |part, index|
+        section = part.split(';')
+        url = section[0][/<(.*)>/,1]
+        name = section[1][/rel="(.*)"/,1].to_sym
+        links[name] = url
+      end if !header_value.nil? && !header_value.empty?
```

Comment:
General ruby style calls for [*not* putting trailing conditional modifiers on multi-line blocks](https://github.com/rubocop-hq/ruby-style-guide#multi-line-if-modifiers). I noted one possible solution above, but you could also use an early return or put the whole `each` block inside of a conditional.